src/content/developers/docs/scaling/optimistic-rollups/index.md

```diff
@@ -1,46 +1,252 @@
 ---
 title: Optimistic Rollups
-description: Introduction to optimistic rollups
+description: An introduction to optimistic rollups as a scaling solution used by the Ethereum community. 
 lang: en
 sidebar: true
 ---
 
+Optimistic rollups are layer 2 (L2) protocols designed to extend throughput of Ethereum's base layer. They reduce computation on the main Ethereum chain by processing transactions off-chain, offering significant improvements in processing speeds. Unlike other scaling solutions, such as [sidechains](/developers/docs/scaling/sidechains/), optimistic rollups derive security from Mainnet by publishing transaction results on-chain. 
```

Comment:
```suggestion
Optimistic rollups are layer 2 (L2) protocols designed to extend the throughput of Ethereum's base layer. They reduce computation on the main Ethereum chain by processing transactions off-chain, offering significant improvements in processing speeds. Unlike other scaling solutions, such as [sidechains](/developers/docs/scaling/sidechains/), optimistic rollups derive security from Mainnet by publishing transaction results on-chain, or [plasma chains](https://github.com/developers/docs/scaling/plasma/), which also verify transactions on Ethereum with fraud proofs, but store transaction data elsewhere.
```

Figured since we referenced validiums on zk-rollup page, worth a similar reference for plasma here :)